xmlsec1/README.md

```diff
@@ -0,0 +1,96 @@
+# xmlsec1
+
+Brief description of xmlsec1
```

Comment:
We should probably populate this with non-generic content.

Personally, for plans that are just libraries and thus won't have any interesting per-package README content, I would prefer no readme at all, but I suppose we would need to change one of the policy documents for that.